packages/python/plotly/plotly/express/__init__.py

```diff
@@ -2,13 +2,24 @@
 `plotly.express` is a terse, consistent, high-level wrapper around `plotly.graph_objects`
 for rapid data exploration and figure generation. Learn more at https://plotly.com/python/plotly-express/
 """
+
 from plotly import optional_imports
 
 np = optional_imports.get_module("numpy")
 if np is None:
     raise ImportError(
         """\
-Plotly express requires numpy to be installed."""
+Plotly Express requires numpy to be installed. You can install numpy using pip with:
+
+$ pip install numpy
+
+Or install Plotly Express and its dependencies directly with:
+
+$ pip install "plotly[express]"
```

Comment:
Previously we settled on `plotly[pandas]` to install Plotly with our recommended version of pandas; do we want to stick with that (`plotly[numpy]`) or make it "the dependencies for Plotly Express"? If we have other dependencies then `plotly[express]` could e.g. require `plotly[numpy]` & whatever else.

I don't have a strong opinion but something to make an active decision on.

Reply:
If I remember correctly you originally proposed `plotly[express]` and I'm the one who argued for `plotly[pandas]` instead 😄 mea cupla

On reflection, and especially since the Plotly Express dependencies have changed following the Narwhals integration, I think it makes more sense to have `plotly[express]` as shorthand for "Plotly, plus whatever dependencies happen to be required by the current version of Plotly Express". That way if the dependencies change in the future we don't have to change the install name.

Reply:
🫡 lol, sounds good to me!